src/filter/index.ts

```diff
@@ -1 +1,2 @@
 export * from './filterDsl'
+export { KeyStorageFacade, StorageFacade, LocalStorageImpl, KeyStorageImpl } from '@icure/api'
```

Comment:
Add this to the index.ts at the project root instead
`src/index.ts` instead of `src/filter/index.ts`